tests/driver_mrf24j40/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = driver_mrf24j40
```

Comment:
Though I appreciate the effort put into this test application, I tend to say we can savely drop it. Testing the driver would simply be:
- build the `examples/gnrc_networking` with `USEMODULE=mrf24j40 BOARD=BLBLBLBL make`
- set/get options (addresses, channel, pan)
- send packets
- done

Or did I miss any additional functionality provided by this test app?